rllib/utils/framework.py

```diff
@@ -258,8 +258,8 @@ def get_variable(value: Any,
 
 
 @Deprecated(
-    old="rllib/models/utils.py::get_activation_fn",
-    new="rllib/utils/framework.py::get_activation_fn",
+    old="rllib/utils/framework.py::get_activation_fn",
+    new="rllib/models/utils.py::get_activation_fn",
```

Comment:
did we really swapped new and old 😆

Reply:
haha, yeah, the args were wrong in the deprecation warning :/